packages/MJServer/src/index.ts

```diff
@@ -171,7 +185,7 @@ export const serve = async (resolverPaths: Array<string>, app = createApp(), opt
     cors<cors.CorsRequest>(),
     BodyParser.json({ limit: '50mb' }),
     expressMiddleware(apolloServer, {
-      context: contextFunction({ setupComplete$, dataSource }),
+      context: contextFunction({ setupComplete$, dataSource, readOnlyDataSource }),
```

Comment:
It seems to have hallucinated a change to the `contextFunction` signature. If it's not updated to actually make use of the new `readOnlyDataSource` and include a `dataSources` value in the context object then the rest of this code will bomb.